src/requirements/data/exams/ExamCredit.ts

```diff
@@ -416,33 +419,46 @@ function userDataToCourses(
       const excludedMajor =
         exam.fulfillment.majorsExcluded && exam.fulfillment.majorsExcluded.includes(major);
       if (!excludedMajor) {
```

Comment:
Can you add a comment that this case is where the AP/IB credit can be used for the major requirements